src/modules/admin/helpers/AdminTableData.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import Avatar from '@material-ui/core/Avatar';
+import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
+
+import Table from 'src/ui/table';
+
+import csx from './AdminTableData.scss';
+
+
+const avatarUrl = 'https://cdn.pixabay.com/photo/2018/04/28/13/18/man-3357275_960_720.png';
+
+const header = ['', 'name', 'description', 'created', 'modified', 'added by', ''];
+
+export const CONFIG: Table.TableConfig = {
+  height: 500,
+  width: 1350,
+  itemSize: 80
+};
+
+const formatValue = (value: string, cutIndex: number): string =>
+  value.length > cutIndex ? value.substr(0, cutIndex - 3) + '...' : value;
+
+export const getTableData = (data) => {
+  // @TODO Add the interface when backend will provide full model
+  const tableData: Table.Row[] = data.map((item) => {
+    const row: Table.Row = {
+      id: {
+        component: <p className={csx.id}>{item.id}</p>
```

Comment:
Maybe instead of object better option will be to use id: component - less boilerplate